tests/h/views/activity_test.py

```diff
@@ -763,11 +772,17 @@ def users(self, request, user, factories):
                 "user": factories.User()}
 
     @pytest.fixture
-    def controller(self, request, group, pyramid_request):
+    def OrganizationResource(self, patch):
+        return patch(
+            'h.views.activity.OrganizationResource')
```

Comment:
I tried to specify the logo="..." here when creating the patch but it didn't work. My theory is that's because the fixture gets created before .start() is called on the patch. This means that I still have to set 
```
        expected_logo = "http://example.com/organizations/pubid/logo"
        controller._org_resource.logo = expected_logo
```
At the beginning of the test which is kinda a pain but I'm not sure if there's anything better?